content/enterprise/operational-modes/index.mdx

```diff
@@ -0,0 +1,162 @@
+---
+page_title: Operational Modes - Terraform Enterprise
+description: >-
+  Learn about the different operational modes of Terraform Enterprise.
+---
+
+# Terraform Enterprise Operational Modes
+
+Before deploying Terraform Enterprise, you must choose an operational mode.
+Each operational mode changes where Terraform Enterprise stores its data.
+
+## Terraform Enterprise Data
+
+Terraform Enterprise data utilizes the following types of data:
+
+- **PostgreSQL Database** - Stateful Terraform Enterprise application data. (e.g. workspace settings, organization settings, run information, user information)
+
+- **Object Storage** - Artifacts generated by Terraform Enterprise. (e.g. state files, plan files, run logs, configuration versions)
+
+- **Vault** - Encryption keys used to encrypt and decrypt objects within object storage.
+
+- **Configuration** - Configuration settings. (e.g. the hostname, object storage credentials, database credentials, concurrency settings, etc.)
+
+## Operational Modes
+
+The following table lists each operational mode and where it stores each type of data.
+
+| Operational Mode                        | PostgreSQL Database                                                              | Object Storage                                                                  | Vault                                                                             | Configuration                              |
+| --------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- | ------------------------------------------ |
+| [Mounted Disk](#mounted-disk)           | Stored in a directory on the instance backed by user-managed persistent storage. | Stored in a directory on the instance backed by user-managed persisent storage. | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
```

Comment:
is there a way to take out the "stored in" in the front of each of these? It'd be easier to parse if we got more to the actual point quickly. So like "Directory on the Instance"  "PostgreSQL database, unless using External Vault." etc. Thoughts?

Reply:
Done!